.github/workflows/job_cxx_unit_tests.yml

```diff
@@ -201,6 +201,12 @@ jobs:
           ${{ env.SETUPVARS_COMMAND }}
           ${{ env.INSTALL_TEST_DIR }}/ov_cpu_unit_tests_vectorized --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-CPUUnitTestsVectorized.xml
 
+      - name: NPU plugin unit tests 
+        if: ${{fromJSON(inputs.affected-components).NPU.test && runner.os == 'Windows'}}
+        run: |
+          ${{ env.SETUPVARS_COMMAND }}
+          ${{ env.INSTALL_TEST_DIR }}/ov_npu_unit_tests --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-NPUUnitTests.xml
```

Comment:
Would it require the actual device in the system? Can we only limit these to NPUW tests?

cc: @PatrikStepan @sbutnari

Reply:
usually not, this is the idea of unit-tests, more over they limited to npuw test cases right now.
@akladiev please also check whether it is enough